natlas-server/app/static/natlas.js

```diff
@@ -110,6 +111,23 @@ $(document).ready(function() {
     }
 });
 
+
+$(document).ready(function() {
+    if (document.getElementById("particles-js")) {
+        function setMotion(e) {
+            if (e.matches) {
+                particlesJS.load("particles-js", '/static/conf/particles-static.json');
```

Comment:
The issue here is that Webpack is not going to be able to process this as is and we won't be able to add an asset hash in the filename to cache bust if we change this. Are we able to load this ourselves or pack it into the bundle?

Reply:
Converted these json files into exported ts constants and then import them and call the function that way. Webpack should pick them up now.